assets/aml-benchmark/components/src/batch_benchmark_score/batch_score/header_handlers/oai/oai_header_handler.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""The class for OSS header handler."""
```

Comment:
nit: OAI header handler, same in the class docstring.

Reply:
changed